test/testUtils/TestSuite.scala

```diff
@@ -72,20 +70,6 @@ trait TestSuite extends AnyWordSpec with MockFactory with GuiceOneAppPerSuite wi
     await(awaited.body.consumeData.map(_.utf8String))
   }
 
-  //noinspection ScalaStyle
```

Comment:
I'm guessing these were unused?

Reply:
Yeah, they were only being used in the unit test but I've deleted that now